main/guides/coreeval/permissions.md

```diff
@@ -0,0 +1,96 @@
+# Coding Permissions
+
+Write a json file with the permissions that the proposal will need. For example,
+[gov-add-psm-permit.json](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/psm/gov-add-psm-permit.json) 
+is the proposal Agoric created for the PSM contract. Note that permissions are set by listing the
+permission and setting it to any String. Thus,`"bankManager": true` and `"bankManager": false` are
+equivalent; they both set the `bankManager` permission.
+
+::: details Show example permissions file
+```
+{
+  "consume": {
+    "agoricNamesAdmin": true,
+    "bankManager": true,
+    "board": true,
+    "chainStorage": true,
+    "zoe": "zoe",
+    "feeMintAccess": "zoe",
+    "economicCommitteeCreatorFacet": "economicCommittee",
+    "econCharterKit": true,
+    "provisionPoolStartResult": true,
+    "psmKit": true,
+    "chainTimerService": "timer"
+  },
+  "produce": {
+    "testFirstAnchorKit": true
+  },
+  "installation": {
+    "consume": {
+      "contractGovernor": true,
+      "psm": true,
+      "mintHolder": true
+    }
+  },
+  "instance": {
+    "consume": {
+      "economicCommittee": true
+    }
+  },
+  "issuer": {
+    "produce": {
+      "DAI": true
+    },
+    "consume": {
+      "DAI": true
+    }
+  },
+  "brand": {
+    "consume": {
+      "DAI": true,
+      "IST": true
+    },
+    "produce": {
+      "DAI": true
+    }
+  }
+}
+```
+:::
+
+## Consume Section
+
+In this section you need to set all the permissions that your contract will need to be able to use
+(i.e., "consume"). Some of the listed permissions in the example PSM permission file above are of
+general interest to most contracts, while others are more specific to the PSM contract.
+
+* **agoricNamesAdmin**: Grants write access to the Agoric name service. This permission is somewhat specific to the PSM contract.
+* **bankManager**: Grants access to bank-related functionality within ERTP, allowing the contract to manipulate nearly all Cosmos assets in the chain. Because this capability is very powerful, this permission should only be granted to contracts that absolutely need it.
+* **board**: Grants write access to the board name service. This permission is somewhat specific to the PSM contract.
```

Comment:
```suggestion
* **board**: Grants write access to the [board name service](https://docs.agoric.com/guides/wallet/#the-agoric-board).
```

That is: (1) the board isn't specific to the PSM and (2) we have a section that documents it; let's refer to it. (I might not be using the right URL convention)

Reply:
Fixed. 

Also, although your URI was well-formed & valid, I converted it to the internal-reference format that the other URI in the docs use. (i.e. the URI is now /guides/wallet/README.md#the-agoric-board)